packages/generators/src/index.js

```diff
@@ -143,9 +143,9 @@ const libTreeItems = [
     [
         'viz',
         [
-            ['stack', [['chart'], ['xAxis'], ['yAxis'], ['layers']]],
+            ['stack', [['cchart'], ['xAxis'], ['yAxis'], ['layers']]],
```

Comment:
Yeah, it was throwing duplicate key warnings since the name of these are the id of the node, which is used for the key. I could update the key to include the index as well.

Reply:
No problem, thank you for the explanation, I wasn't sure about the reason, no need to change.